images/router/f5/pod.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I can remove it; I snabbed it from https://github.com/pweil-/origin/commit/fa89f112c1263324de59b4661389a5c6606904fb